### PR TITLE
Update sensor.miflora.markdown

### PR DESCRIPTION
**Pull request by themanieldaniel**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by themanieldaniel** [comment]
This is just meant to be a grammar and formatting update. I don't think I've changed the meaning of anything.

File: source/_components/sensor.miflora.markdown [hunk 1]
```diff
@@ -13,51 +13,25 @@ ha_release: 0.29
...
     mac: 'xx:xx:xx:xx:xx:xx'
+    name: Flower 1
+    force_update: false
+    median: 3
     monitored_conditions:
+      - moisture
+      - light
       - temperature
+      - conductivity
+      - battery
```
Comment (fabaff) [inline source/_components/sensor.miflora.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps users to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_components/sensor.miflora.markdown [hunk 2]
```diff
@@ -75,23 +49,39 @@ sensor:
 - **cache_value** (*Optional*): Define cache expiration value in seconds (defaults to 1200 if not defined)
 - **adapter** (*Optional*): Define the Bluetooth adapter to use (defaults to hci0). Run `hciconfig` to get a list of available adapters.
 
-Note that by default the sensor is only polled once every 20 minutes. This means with the `median: 3` setting will take as least 40 minutes before the sensor will report a value after a Home Assistant restart. As the values usually change very slowly, this isn't a big problem.
-Reducing polling intervals will have a negative effect on the battery life.
+NOTE By default the sensor is only polled once every 20 minutes. So, if you set `median: 3` it will take _at least_ 40 minutes before the sensor will report a value after a Home Assistant restart. Since the values usually change very slowly, this usually isn't a big problem. Keep in mind though that reducing polling intervals will have a negative effect on the battery life.
```
Comment (fabaff) [inline source/_components/sensor.miflora.markdown]: if you want a note then it should be formatted as [note](https://www.home-assistant.io/developers/documentation/create_page/#html).

**Review by fabaff — DISMISSED** [review]
From a user point of view does it makes more sense to have the  details to get the data for the configuration in Home Assistant before the configuration sample.

**Comment by themanieldaniel** [comment]
All good points @fabaff 

On my initial reading of the documentation I thought that scanning for a MAC address was not needed on hass.io so I thought it might be less clumsy for hass.io users to have the configuration at the top. I was probably overthinking it (or underthinking it).

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thank you for updating this PR @themanieldaniel! 👍 
IMHO this one can be merged as soon as the build passes (I've made a tiny tiny tweak).

**Pull request merged**